aequilibrae/transit/gtfs_loader.py

```diff
@@ -90,26 +93,28 @@ def __load_date(self):
         self.logger.debug("Starting __load_date")
         self.zip_archive = zipfile.ZipFile(self.archive_dir)
 
-        self.signal.emit(["start", 7, "Loading routes"])
+        self.signal.emit(["start", 0, 7, "Reading GTFS data", "master"])
```

Comment:
Do we have two progress bars in QGIS?

Reply:
In two screens: traffic assignment and GTFS importer. Do you want to remove them and make them only one?